packages/earl/src/errors/stack-traces.test.ts

```diff
@@ -41,6 +42,6 @@ describe('stack traces for errors', () => {
       return nestedGetControl()
     }
     const control = nestedValidator()
-    expect(control.file).to.equal(import.meta.url)
+    expect(control.file).to.equal(fileURLToPath(import.meta.url))
```

Comment:
Just a fix. I believe this changed in node 18/20, but `import.meta.url` is a `file://` URL, and `control.file` is a path.